Bangazon/Views/Products/Create.cshtml

```diff
@@ -32,7 +32,7 @@
                 <input asp-for="Quantity" class="form-control" />
                 <span asp-validation-for="Quantity" class="text-danger"></span>
             </div>
-            <div class="form-group">
+            <div class="form-group hidden">
                 <label asp-for="UserId" class="control-label"></label>
                 <select asp-for="UserId" class ="form-control" asp-items="ViewBag.UserId"></select>
```

Comment:
Do not user `ViewBag`. If you need to get data into the view that is not in the Model, you need to use a View Model.